src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/api/HandEventHandlerTest.java

```diff
@@ -184,4 +186,39 @@ void play_one_card_with_announcing_score_and_two_matches() {
     assertEquals(MatchState.ANNOUNCING, actuelMatches.get(0).getMatchState());
     assertEquals(MatchState.PLAYING, actuelMatches.get(1).getMatchState());
```

Comment:
Was ist das für ein Usecase, wo es in einem Spiel einen Match im Status ANNOUNCING und einen im Status PLAYING gibt?

Reply:
Kann man auch umdrehen ist, aber anscheinend auch auf diese Richtung möglich. Für ein besseres Verständnis unseres Programms.

Reply:
war schon vorher so, fixen wir nicht in diesem PR